pkg/distributor/distributor_test.go

```diff
@@ -2654,30 +2655,37 @@ func TestSortLabels(t *testing.T) {
 }
 
 func TestDistributor_Push_Relabel(t *testing.T) {
-	ctx := user.InjectOrgID(context.Background(), "user")
+	const orgID = "user"
+	ctx := user.InjectOrgID(context.Background(), orgID)
```

Comment:
why was this change made?

Reply:
I was using this const down the test in during an exploration of mine, forgot to undo this change. Now it is undone.